system-variables.md

```diff
@@ -275,6 +275,7 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - This variable is used to set the concurrency of the `scan` operation.
 - Use a bigger value in OLAP scenarios, and a smaller value in OLTP scenarios.
 - For OLAP scenarios, the maximum value cannot exceed the number of CPU cores of all the TiKV nodes.
+- If the tables have quite a lot partitions, reduce the value appropriately to avoid TiKV becoming OOM.
```

Comment:
```suggestion
- If a table has a lot of partitions, you can reduce the variable value appropriately to avoid TiKV becoming Out of Memory (OOM).
```

Reply:
Seems like it is unnecessary to uppercase "out of memory" here？It is used much more as a part of the sentence, not a proper noun.

Reply:
OK, comment changed,